client/my-sites/site-settings/podcasting-details/style.scss

```diff
@@ -0,0 +1,16 @@
+.podcasting-details__wrapper .form-setting-explanation {
+	margin-bottom: 6px;
+}
+
+.podcasting-details__category-selector {
+	box-sizing: border-box;
+	float: left;
+	width: 35%;
+}
+
+.podcasting-details__basic-settings {
+	box-sizing: border-box;
+	float: left;
+	width: 64%;
+	padding-left: 24px;
```

Comment:
We should put right-padding on the category selector instead, otherwise at smaller breakpoints it ends up looking like this:

![image](https://user-images.githubusercontent.com/363749/37941410-aa9abf02-3133-11e8-841c-6776800f619d.png)